core/src/test/scala/com/mesosphere/usi/core/SchedulerTest.scala

```diff
@@ -101,4 +105,55 @@ class SchedulerTest extends AkkaUnitTest with Inside {
     output.cancel()
     mesosCompleted.futureValue shouldBe Done
   }
+
+  "Persistence flow pipelines writes" in {
```

Comment:
After reading this test several times, I'm inclined to submit that we should just delete this test. I don't think that it's testing anything meaningful; it appears to be testing Akka itself, which already has tremendous test coverage.

Perhaps a test that would be more meaningful would be a unit test for the persistence flow itself which asserts that delete and create operations are applied in order, and that the ordinary of elements being processed is preserved (perhaps some fuzzing can be applied by causing the storage operations to complete in a random number of milliseconds time)

Reply:
I agree that this test does not create much value. It was useful when i had a custom [graph stage logic](https://github.com/mesosphere/usi/blob/92c1c088336351b4fa3375b1bc009e97cc082968/core/src/main/scala/com/mesosphere/usi/core/Scheduler.scala#L166-L217) but since we got rid of that, we don't need this exact test anymore. I have rewritten the test using `Thread.sleep` to apply "fuzzing", lmk if that is not ideal.